src/utilities/createRouter.ts

```diff
@@ -43,38 +44,16 @@ export function createRouter<T extends Routes>(routes: T, options: RouterOptions
     Object.assign(route, newRoute)
   }
 
-  function pushUrl(url: string, options: RouterPushOptions = {}): Promise<void> {
-    return navigation.update(url, options)
+  function replace(url: string, options: RouterReplaceOptions = {}): Promise<void> {
+    return push(url, { ...options, replace: true })
```

Comment:
Updated `replace` to just call `router.push`. That way theres only one implementation of what "replace" means.